app/windows/Storage/Components/StorageElement.jsx

```diff
@@ -75,16 +83,19 @@ class StorageElement extends React.Component {
   }
 
   render() {
-    if (!this.props.storageStore) return <tr />
     const el = this.props.element
     return (
       <tr key={el.hash} onContextMenu={this._handleContextMenu.bind(this)}>
+
+        { this.props.storageStore ?
         <td>
           <input
             onClick={this._handleCheckboxOnClick.bind(this, el)}
             type='checkbox'
           />
         </td>
+        : <td></td> }
```

Comment:
Maybe we can return `null`, to not have have an empty column, same with the header

Reply:
If I return null, the CSS rules are broken due to the `-nth` breaking most of the content display, so providing an empty `<td>` seems to be fine.

Reply:
Oh, I see

Reply:
Good point though :) I am glad that you caught that!